src/RPC.js

```diff
@@ -186,7 +186,7 @@ class RPC {
             let valueForOwner = { "CurveSpec": curve, "Account": address, "Minvalue": minValue, "PaginationToken": "" }
             let runningUtxos = [];
             let runningTotal = BigInt("0");
-            while (true) {
+            while (constant.ReqCondition) {
```

Comment:
`constant.ReqCondition` here addresses the rule https://eslint.org/docs/rules/no-constant-condition

Reply:
Lets set the options to not check loops {checkLoops: false}. Reading this without immediately thinking of ESLint rules makes it seem like this might change or be a variable, when it is supposed to just be true.

Reply:
Open to @adrianbona @rfrabasile 's opinions on this as well, I just find it a confusing read. I immediately know what true is.

Reply:
I don't believe we are gonna have a huge amount of loops like this one. I'm always up to having constants for everything but I can live with an IF TRUE. It might be easier to understand reading it in the original way.